1.1.3

brianmcmichael opened the pull request:
1.1.3

Kovan and Mainnet

* Adds address for `MCD_JOIN_ETH_B`
* Adds address for `MCD_FLIP_ETH_B`

zdumitru commented on releases/kovan/active:
```diff
@@ -1 +1 @@
-1.1.2
\ No newline at end of file
+1.1.3
```
Didn't we agree that the active link is updated only after the spell is cast?

zdumitru replied:
Ah, that's kovan. All good then.

EdNoepel replied:
For Kovan though?

brianmcmichael replied:
Ah, I just updated the symlink for completeness. I am pretty sure I updated mainnet also. i can undo it, but we'll need to remember to update this once it's cast.

zdumitru reviewed (changes requested):
Please revert mainnet back to 1.1.2 and I will remember to update it to 1.1.3 when (if) the voting approves it and the spell is cast.

brianmcmichael commented:
* reverted mainnet `active` to 1.1.2 for now.
* added `CHANGELOG` address to kovan and mainnet 1.1.3
* Updated IPFS links for new json
* Added ChainLog.abi to kovan and mainnet zip files

brianmcmichael commented on releases/mainnet/active:
```diff
@@ -1 +1 @@
-1.1.2
\ No newline at end of file
+1.1.3
```
Updated.

kmbarry1 reviewed (approved):
LGTM

At some point we should standardize on a canonical ilk ordering (probably alphabetical). Hopefully no one depends on the order they appear in the changelog...
